lib/currency.rb

```diff
@@ -0,0 +1,29 @@
+module Currency
+  extend self
+  
+  def args(*currency)
+    currency = currency.flatten.compact
+    return if currency.blank?
+    case currency.length
+    when 0
+      nil
+    when 1
+      arg = currency[0]
+      return if arg.blank?
```

Comment:
`return` should be avoided in general. Specifically from inside deep `if`/`case`.